.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
This file is temporary. I had to fork dependencies and link them from local directory.